src/views/JoinHouse.vue

```diff
@@ -65,7 +66,10 @@ export default {
 data(){
     return {
      roomkey: undefined,
-     homename: undefined
+     homename: undefined,
+     rules: {
+         min: v => v.length === 32 || 'House keys contains 32 characters'
```

Comment:
Should be "House keys contain 32 characters". Also 'min' might should be 'minLength'